base_multi_company/hooks.py

```diff
@@ -29,8 +29,10 @@ def set_security_rule(env, rule_ref):
     rule = env.ref(rule_ref)
     if not rule:  # safeguard if it's deleted
         return
+    # We don't want to enable the rule if it isn't already active, as this leads
+    # to subsequent test fails in a CI context. This should be done via the core
+    # res.config.settings methods
     rule.write({
```

Comment:
I think you should activate it if you are not in test environment:
```suggestion
    from odoo.tools import config
    if not config["test_enable"]:
```